x-pack/packages/kbn-elastic-assistant-common/tsconfig.json

```diff
@@ -21,5 +21,6 @@
     "@kbn/core",
     "@kbn/actions-plugin",
     "@kbn/logging-mocks",
+    "@kbn/elastic-assistant",
```

Comment:
nit: is it possible to share this helper in a way that doesn't require a new reference?

Reply:
oh man, i dont understand how this got added...

Reply:
i see... i put the tests on the wrong helpers.test.ts... whoops. fixing!